safe_transaction_service/history/models.py

```diff
@@ -1806,6 +1806,15 @@ def get_delegates_for_safe_and_owners(
             .distinct()
         )
 
+    def remove_delegates_for_owner_in_safe(
+        self, safe_address: ChecksumAddress, owner_address: ChecksumAddress
+    ) -> int:
```

Comment:
Add documentation for what you are returning, as that `.delete()[0]` is not very intuitive. Also it's good to explain that this method will only delete a delegate if both delegate and Safe match